README.md

```diff
@@ -357,11 +357,11 @@ For this case it is possible to reduce the number of nodes to be contacted to fo
 
 You do however want to make sure that you have some sort of 'random' selection of which 12 out of the 16 nodes will participate in every lock. See [here](https://gist.github.com/fwessels/dbbafd537c13ec8f88b360b3a0091ac0) for some sample code that could help with this.
 
-### Scale beyond 16 nodes?
+### Scale beyond 32 nodes?
 
 Building on the previous example and depending on how resilient you want to be for outages of nodes, you can also go the other way, namely to increase the total number of nodes while keeping the number of nodes contacted per lock the same.
 
-For instance you could imagine a system of 32 nodes where only a quorom majority of `9` would be needed out of `12` nodes. Again this requires some sort of pseudo-random 'deterministic' selection of 12 nodes out of the total of 32 servers (same [example](https://gist.github.com/fwessels/dbbafd537c13ec8f88b360b3a0091ac0) as above). 
+For instance you could imagine a system of 32 nodes where only a quorum majority of `9` would be needed out of `12` nodes. Again this requires some sort of pseudo-random 'deterministic' selection of 12 nodes out of the total of 32 servers (same [example](https://gist.github.com/fwessels/dbbafd537c13ec8f88b360b3a0091ac0) as above).
```

Comment:
This probably should talk about 64 nodes.

Reply:
This is simply an example we don't have to talk about 64 nodes @kannappanr - there is no such requirement. 16 is much valid as 32.  Regarding limits we already state that elsewhere.

Reply:
ok

Reply:
Fixed and updated @kannappanr